### PR TITLE
chore: use Metadata as payload to create identity

### PR DESCRIPTION
**Pull request by NunoAlexandre**
The REST endpoint that handles this request expects a `Metadata`, not a
type that contains a passphrase. Therefore, we drop `CreateInput` in
favour of a new `Metadata` struct.

Signed-off-by: Nuno Alexandre <hi@nunoalexandre.com>

### CODE REVIEW

**Comment by FintanH** [comment]
Something I'm wondering at this stage is whether these inputs should just be the `PersonPayload` which can include the extensions such as the ethereum address. 

I think that's eventually what that endpoint will be expecting in the future anyway. 

Food for thought :)

File: ui/src/identity.ts [hunk 1]
```diff
@@ -29,13 +31,8 @@ export interface Identity {
 const creationStore = remote.createStore<Identity>();
 export const store = creationStore.readable;
 
-interface CreateInput {
-  handle: string;
-  passphrase: string;
```
Comment (FintanH) [inline ui/src/identity.ts:34]: I'm presuming the `passphrase` was un-needed? What the hell is it doing there in the first place?! :joy:

**Review by FintanH — APPROVED** [review]

File: ui/src/identity.ts (hunk 1 above)
Comment (NunoAlexandre) [reply ui/src/identity.ts:34]: that's right. I don't know :shrug: I tested the whole onboarding flow to be super sure. but if you follow the code you see that we first call the endpoint that creates a keystore and then we call the endpoint that creates the identity.

Comment (xla) [reply ui/src/identity.ts:34]: It used to be a single call, was changed in multiple iterations of the session feature.

**Comment by NunoAlexandre** [comment]
Merged in 773ca6b5e3383f90c2d8b35d19e43b487080ce19

**Pull request closed without merging**